book/integration-service.md

```diff
@@ -137,6 +137,11 @@ Following the [annotation guidelines](https://docs.google.com/document/d/1gyXM3p
 ```
 The Integration service will copy the annotations and labels from the Build PipelineRun and append those to the Test PipelineRuns for traceability across the system per [Labels and Annotations for StoneSoup pipelines](https://docs.google.com/document/d/1fJq4LDakLfcAPvOOoxxZNWJ_cuQ1ew9jfBjWa-fEGLE/edit#) and [StoneSoup builds and tests PRs](https://docs.google.com/document/d/113XTplEWRM63aIzk7WwgLruUBu2O7xVy-Zd_U6yjYr0/edit#)
 
+Below Label provide users an option whether an IntegrationTestScenario will be taken into account when determining if the Snapshot has passed testing. It will be copied to the subsequence Test PipelineRuns. 
```

Comment:
typo: subsequence -> subsequent

Reply:
Also my suggestion is to change `Below Label provide users an option` into `The label below provides users with an option`.

Reply:
and, I suggest modifying `whether an IntegrationTestScenario will be taken into account when determining if the Snapshot has passed testing.` to `whether the results of an IntegrationTestScenario will be taken into account when determining if the Snapshot has passed all required testing.`

Reply:
@ralphbean , I changed it  to " whether the result of a pipelineRun created according the IntegrationTestScenario will be taken into account when determining if the Snapshot has passed all required testing" by referring to your comment. Is it OK for you?

Reply:
Yes, thank you!